qemu/vm.sh

```diff
@@ -8,7 +8,7 @@ kernelargs=""
 bridge=zos0
 graphics="-nographic -nodefaults"
 smp=1
```

Comment:
why changing this file?
looks like need separate PR for this

Reply:
this was by mistake, removed it